app/models/batch.rb

```diff
@@ -0,0 +1,15 @@
+class FermentValidator < ActiveModel::Validator
```

Comment:
Checkout https://guides.rubyonrails.org/active_record_validations.html#inclusion. That will save you some code